src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -0,0 +1,184 @@
+import _ from "lodash"
+import SharedService from "../services/SharedService"
+import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import { pointSub } from "../helpers/point"
+
+export default class PencilMouseHandler extends MouseHandler {
+  constructor(emitter) {
+    super(emitter, [defaultActionFactory, actionFactory], getCursor)
+  }
+}
+
+function actionFactory(e, ctx) {
+  const cpos = getLocal(e)
+  const rects = ctx.getEventsUnderPoint(cpos.x, cpos.y)
+
+  if (e.nativeEvent.button != 0) {
+    return null
+  }
+
+  if (rects.length > 0) {
+    const note = rects[0]
+    if (e.nativeEvent.detail == 2) {
+      return removeNoteAction(ctx, note)
+    } else {
+      const local = pointSub(cpos, note)
+      const type = getDragPositionType(local.x, note.width)
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->